pylua/network.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import socket
+import tornado.httpserver
+import tornado.websocket
+import tornado.ioloop
+import tornado.web
+
+#websocket-client
+from websocket import create_connection
+
+class LuaWebSocketClient(object):
```

Comment:
@OHerasym, rename LuaWebSocketClient -> WebSocketClient. same for server. This is just python code and it's not aware of lua